stimela/kitchen/recipe.py

```diff
@@ -1,27 +1,25 @@
-from cmath import exp
-from multiprocessing import cpu_count
-import os, os.path, re, logging, fnmatch, copy, time
+import os, os.path, re, fnmatch, copy, traceback
```

Comment:
SAD

Reply:
["The Ten Commandments. She was a great believer in some of them."](https://libquotes.com/joe-orton/quote/lbv2c5y)

That's me and PEP-8!